internal/parsing/parse.go

```diff
@@ -134,7 +138,23 @@ func parseWith(file fs.File, parsers []Parser, groupNumber int, log *zap.Sugared
 	}
 
 	finalResults := parsedTestResults[0]
-	log.Debugf("%T was ultimately responsible for parsing the test results", firstParser)
+	cfg.Logger.Debugf("%T was ultimately responsible for parsing the test results", firstParser)
+
+	testIDsAreUnique, err := checkIfTestIDsAreUnique(finalResults, cfg)
+	if err != nil {
+		return nil, err
+	}
+	if !testIDsAreUnique {
+		duplicateTestIDWarningMsg := fmt.Sprintf(
+			"Test result file %q contains two or more tests that share the same metadata (such as name or identifier). %s",
+			file.Name(),
+			"Please make sure test identifiers are unique.",
+		)
```

Comment:
Is there any chance we could get a list of the tests that have issues in here? Can totally be a follow up